MetadataProcessor.Shared/Tables/nanoMemberReferenceTable.cs

```diff
@@ -0,0 +1,165 @@
+        /// </param>
+        public nanoMemberReferencesTable(
+            IEnumerable<MemberReference> items,
+            nanoTablesContext context)
+            : base(items, new MemberReferenceComparer(), context)
+        {
+        }
+
+        /// <summary>
+        /// Gets member reference ID if possible (if method is external and stored in this table).
+        /// </summary>
+        /// <param name="memberReference">Member reference metadata in Mono.Cecil format.</param>
+        /// <param name="referenceId">Member reference ID in .NET nanoFramework format.</param>
+        /// <returns>Returns <c>true</c> if reference found, otherwise returns <c>false</c>.</returns>
+        public bool TryGetMemberReferenceId(
+            MemberReference memberReference,
+            out ushort referenceId)
+        {
+            return TryGetIdByValue(memberReference, out referenceId);
+        }
+
+        /// <inheritdoc/>
+        protected override void WriteSingleItem(
+            nanoBinaryWriter writer,
+            MemberReference item)
+        {
+            if (!_context.MinimizeComplete)
+            {
+                return;
+            }
+
+            var writerStartPosition = writer.BaseStream.Position;
+
+            ushort tag;
+            ushort signature = 0;
+            ushort referenceId = 0;
+
+            if (item is MethodDefinition &&
+                _context.MethodDefinitionTable.TryGetMethodReferenceId(item as MethodDefinition, out referenceId))
+            {
+                // MemberRefParent tag is 3 (MethodDef)
+                tag = 3;
+
+                //
+
+
+                signature = 0;
+            }
+            else if (_context.TypeReferencesTable.TryGetTypeReferenceId(item.DeclaringType, out referenceId))
+            {
+                // MemberRefParent tag is 1 (TypeRef)
+                tag = 1;
+
+                // get signature index
+                signature = _context.SignaturesTable.GetOrCreateSignatureId(item.DeclaringType);
+            }
+            else if (_context.TypeSpecificationsTable.TryGetTypeReferenceId(item.DeclaringType, out referenceId))
+            {
+                // MemberRefParent tag is 4 (TypeSpec)
+                tag = 4;
+
+                // get signature index
+                signature = _context.SignaturesTable.GetOrCreateSignatureId(item.DeclaringType);
+            }
+            else if (_context.TypeDefinitionTable.TryGetTypeReferenceId(item.DeclaringType.Resolve(), out referenceId))
+            {
+                // MemberRefParent tag is 0 (TypeDef)
+                tag = 0;
+
+                // get signature index
+                signature = _context.SignaturesTable.GetOrCreateSignatureId(item.DeclaringType.Resolve());
+            }
+            //else if (_context.TypeDefinitionTable.TryGetTypeReferenceId(item.DeclaringType.Resolve(), out referenceId))
```

Comment:
is this commented on purpose?

Reply:
probably from a merge operation that was a duplicated snnipet...